services/account_last_login.go

```diff
@@ -0,0 +1,18 @@
+package services
+
+import (
+	"github.com/keratin/authn-server/data"
+	"github.com/pkg/errors"
+)
+
+func LastLoginUpdater(store data.AccountStore, accountID int) error {
+	account, err := store.Find(accountID)
```

Comment:
i'm wondering if we can optimize this down to a single query, from `SELECT` + `UPDATE` to simply `UPDATE`.

* mysql should return the number of affected rows
* postgresql can do something similar with `RETURNING`
* sqlite probably requires a second query

so maybe each store should be responsible for returning true/false if the operation was successful, in addition to any possible err?

```go
type AccountStore interface {
	SetLastLogin(id int) (bool, error)
}
```